app/views/layouts/application.html.erb

```diff
@@ -85,6 +85,13 @@
     </style>
   <% end %>
 
+  <script type="text/javascript"  nonce="<%= content_security_policy_script_nonce %>">
+  window.AppcuesSettings = {
+    enableURLDetection: true
+  };
+</script>
+<script type="text/javascript" nonce="<%= content_security_policy_script_nonce %>" src="//fast.appcues.com/119641.js"></script>
```

Comment:
Blocker: this presumably came from upstream docs, but [protocol-relative URLs are a security problem](https://www.paulirish.com/2010/the-protocol-relative-url/), because [they enable man-on-the-side attacks](https://www.netresec.com/?page=Blog&month=2015-03&post=China%27s-Man-on-the-Side-Attack-on-GitHub).  SCP always serves content over HTTPS and Appcues is always available over HTTPS, so protocol-relative URLs are unnecessary here, and we should explicitly specify "https".

```suggestion
<script type="text/javascript" nonce="<%= content_security_policy_script_nonce %>" src="https://fast.appcues.com/119641.js"></script>
```